etl/steps/data/garden/un/2023-08-29/long_run_child_mortality.meta.yml

```diff
@@ -0,0 +1,96 @@
+dataset:
+  title: Long-run child mortality (Gapminder; UN IGME)
```

Comment:
We are moving away from adding producer names in dataset titles, but for now you can keep it if you think it's convenient.